stardog/admin.py

```diff
@@ -1069,7 +1069,7 @@ class StoredQuery(object):
         https://www.stardog.com/docs/#_managing_stored_queries
     """
 
-    def __init__(self, name, client):
+    def __init__(self, name, client, details={}):
```

Comment:
The default value should be `None` since you should not use mutable objects in kwargs. You are not using it directly but it is good practice to avoid it since the default will get mutated causing strange bugs.

Reply:
> 

I probably wont finish this PR since I accepted an offer elsewhere. Feel free to close this or pull it and finish the issue  :)